qemu/tests/cfg/win_virtio_single_driver.cfg

```diff
@@ -0,0 +1,50 @@
+-win_virtio_single_driver:
+    type = win_virtio_single_driver
```

Comment:
Suggest to improve the name of the case, didn't get what you want to test from the name